.template/variants/web/package.json.rb

```diff
@@ -37,9 +37,16 @@
   '--no-source-map',
   '--load-path=node_modules'
 ]
+production_bundled_stylesheet_options = bundled_stylesheet_options + [
+  '--style=compressed'
+]
 
 run %(npm set-script build "node app/javascript/build.js")
 run %(
   npm set-script build:css \
```

Comment:
It would be best to use a prefix here and avoid it below:

```
npm set-script build:css-production \
```

So `production` is special, while `dev` is the base/standard mode.

Reply:
Thanks for the suggestion! The default one should be the `development` environment.

I've updated it in 2c9789c ✅

Reply:
The `build:css-production` won't be trigger as no one call that, the https://github.com/rails/cssbundling-rails/blob/main/lib/tasks/cssbundling/build.rake#L20 enhance the `css:build` by default, we need to go with the Sang's original solution FYI @olivierobert I will open a PR for that cc @sanG-github 

<img width="623" alt="image" src="https://github.com/nimblehq/rails-templates/assets/11751745/199a8b88-ad30-4614-b487-8b6ab45ea795">